src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,74 @@
+import { useMemo } from 'react';
+import Link from 'next/link';
+
+import config from '../../../package.json';
+import { Helmet } from 'react-helmet';
+
+import { GrPrevious as PreviousIcon, GrNext as NextIcon } from 'react-icons/gr';
+import styles from './Pagination.module.scss';
+
+const MAX_NUM_LINKS = 10;
+
+const { homepage = '' } = config;
+
+const Pagination = ({ pagesCount, currentPage, basePath }) => {
+  const path = `${basePath}/page/`;
+
+  const hasPreviousPage = pagesCount > 1 && currentPage > 1;
+  const hasNextPage = pagesCount > 1 && currentPage < pagesCount;
+
+  const pages = useMemo(() => {
+    let pages = [];
+    for (let i = 1; i < pagesCount + 1; i++) {
+      pages.push(i);
+    }
+    return pages;
+  }, [pagesCount]);
+
+  return (
+    <>
+      <Helmet>
+        {!hasPreviousPage && <link rel="canonical" href={`${homepage}${basePath}`} />}
+        {hasPreviousPage && <link rel="prev" href={`${homepage}${path}${currentPage - 1}`} />}
+        {hasNextPage && <link rel="next" href={`${homepage}${path}${currentPage + 1}`} />}
+      </Helmet>
+
+      <nav className={styles.nav}>
```

Comment:
is it okay to semantically have 2 nav elements on a page? since we're using it for the top level nav too? not sure, might be worth double checking

Reply:
Not sure also...I have search a little and seems ambiguous. As definition the nav element is intended for the **primary** navigation block of the page. Pagination links would fit , but as you say the menu is already there and it is a higher navigation block...

But for example here, in the second example, is using two nav elements:  https://www.w3.org/TR/2011/WD-html5-author-20110809/the-nav-element.html

Maybe in case of doubt I can leave it as `div`, what do you think?

Reply:
seems like it would be okay to leave it, i dont feel strongly about it as long as it seems like w3 is doing it safely